k8s/sample-pv-yamls/blockdeviceclaim.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: openebs.io/v1alpha1
+kind: BlockDeviceClaim
+metadata:
+  name: test-blockdeviceclaim
+  namespace: openebs
+spec:
```

Comment:
Let us try to put comments for end user  understanding

Reply:
Done